AraEvent/AraEventCalibrator.cxx

```diff
@@ -19,20 +20,58 @@
 #include <zlib.h>
 #include <cstdlib>
 #include <sstream>
+#include <numeric>
+
+//! Imported from the conditioner class, 27-09-2021 -MK-
```

Comment:
This is really good, but also, a sentence saying _what_ it does. E.g. "Returns if a calibration type should or should not trim the first block of a waveform. I know I'm being picky, but if we don't do the docs now, we'll never do them :)

Reply:
No problem. Let me work on it